.circleci/config.yml

```diff
@@ -119,6 +119,13 @@ aliases:
       name: Disable Xdebug for built-in web tests
       command: test -e /usr/local/etc/php/conf.d/docker-php-ext-xdebug.ini && sudo rm -f $_ || true
 
+  - &STEP_INSTALL_DOCKERIZE
```

Comment:
@SammyK I am explicitly installing dockerize here as I think we pushed a version of the containers without it.
In the future we might want to properly install all the tools in the images when we refactor all our dev machines.